src/libraries/System.Globalization/tests/Invariant/InvariantMode.cs

```diff
@@ -416,14 +455,17 @@ public static IEnumerable<object[]> ToUpper_TestData()
             yield return new object[] { "embedded\0NuLL\0Byte\0", "EMBEDDED\0NULL\0BYTE\0", true };
 
             // LATIN SMALL LETTER O WITH ACUTE, which has an upper case variant.
-            yield return new object[] { "\u00F3", "\u00D3", false };
+            yield return new object[] { "\u00F3", "\u00D3", true };
 
             // SNOWMAN, which does not have an upper case variant.
             yield return new object[] { "\u2603", "\u2603", true };
 
             // RAINBOW (outside the BMP and does not case)
```

Comment:
Should we fix this comment while we're here? "Rainbow" doesn't case because it's not an alphabetic character. Being outside the BMP isn't important. In fact, the test you added immediately after this is a character outside the BMP and _does_ change case. :)